odl/contrib/mrc/README.md

```diff
@@ -0,0 +1,26 @@
+# MRC
```

Comment:
I guessed as much. Perhaps there is a open dataset somewhere and you could add it to datasets and make a proper example?